.github/workflows/tests.yml

```
@@ -0,0 +1,16 @@
+name: Run tests
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: pip install -r data_collection/requirements.txt
```

Using cache could improve rerunning CI in the same PR. It should be tested if that's the case for us, though :)

```suggestion
      - name: Cache pip
        uses: actions/cache@v2
        with:
          path: ~/.cache/pip
          key: ${{ runner.os }}-pip-${{ hashFiles('data_collection/requirements.txt') }}
          restore-keys: |
            ${{ runner.os }}-pip-
      - name: Install dependencies
        run: pip install -r data_collection/requirements.txt
```